index.php

```diff
@@ -74,6 +75,9 @@
 
 	Config::init();
 	Session::init();
+	$trace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS, 1);
+	$command = basename($trace[0]['file']);
+	Process::start($command, Process::PROC_APP);
```

Comment:
I'm a unsure with this part.

Is the process of the "Frontend-APP" the php-fpm process or the request of the user? How do you see that?
Will `getmypid()` return the same PID for each request from the same php-fpm process or is each request a unique PID?

Reply:
There can be multiple `php-fpm` processes per pool, so `getmypid()` will return the specific PID of the current `php-fpm` spawn processing the request.

Reply:
Hm.. I think I will remove this part..

My goal was to see the whole System processes in the process table as a kind of monitoring for the admin page later, but I have the fear to make it to complicated

Reply:
Why would it be too complicated? If each Friendica process can self-declare itself in the process table, then what's the matter?

Reply:
Because the question is how the process will get removed after the call. Hm... But maybe a simple Process::stop at the end should be enough.. I've to test it..